apps/online_oidc_provider/claims.py

```diff
@@ -42,3 +42,13 @@ class Onlineweb4ScopeClaims(ScopeClaims):
 
     def scope_onlineweb4(self):
         return Onlineweb4Userinfo(self.user).oidc()
+
+    info_nibble = (
+        _("Nibble"),
+        _(
+            "Informasjon om dine kjøp og saldo på Nibble",
+        ),
+    )
+
+    def scope_nibble(self):
+        return {'test':'this data was returned'}
```

Comment:
I am highly uncertain whether this is necessary or not, and if it is: what it really does..

Reply:
Learned recently that an empty dictionary should be the default value. Else, "test" is added as a claim to /userinfo with the returned value.